src/poke_env/environment/target.py

```diff
@@ -0,0 +1,57 @@
+"""This module defines the Target class, which represents the types of
+targets a move can have
+"""
+
+import logging
+from enum import Enum, unique, auto
+
+# This is an enum for all the targets you can have
+@unique
+class Target(Enum):
+    """Enumeration, representing targets for each move
+    in a battle."""
+
+    UNKNOWN = auto()
```

Comment:
do we expect any unknown value? if not, it might be better to raise an exception

Reply:
This is a good callout. Will change!